src/vuln_analysis/utils/functions_parsers/c_lang_function_parsers.py

```diff
@@ -517,7 +519,9 @@ def search_for_called_function(
         type_documents: list[Document],
         callee_function_file_name: str,
         fields_of_types: dict[tuple, list[tuple]],
-        functions_local_variables_index: dict[str, dict]
+        functions_local_variables_index: dict[str, dict],
+        documents_of_functions: list[Document],
+        type_inheritance: dict[Tuple[str, str], List[Tuple[str, str]]]
```

Comment:
@tmihalac What about assigning default empty/None value to new unused arguments/parameter s `type_inheritance` and `documents_of_functions`?